<html>
    <head>
        <title>miRvestigator Framework: Detect the miRNAs Driving Co-Expression Signatures</title>
        <meta name="copyright" content="&copy; 2010 Institute for Systems Biology" />
        <style> body { font-family: arial, sans-serif; } </style>
        <script type="text/javascript">
          var _gaq = _gaq || [];
          _gaq.push(['_setAccount', 'UA-19292534-1']);
          _gaq.push(['_trackPageview']);

          (function() {
              var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
              ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
              var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
          })();
        </script>
    </head>
    <body bgcolor='#333333'>
        <center>
            <table width="600">
                <tr>
                    <td>
                        <table cellpadding='15%' cellspacing='5' bgcolor='#999966'>
                            <tr>
                                <td align='center' valign='center' bgcolor='#333333' width='80'>
                                    <b><a style='color: rgb(255,0,0); text-decoration:none' href='/'>H</a><a style='color: rgb(204,204,0); text-decoration:none' href='/'>ome</a></b>
                                </td>
                                <td align='center' valign='center' bgcolor='#333333' width='80'>
                                    <b><a style='color: rgb(255,0,0); text-decoration:none' href='/check_status'>R</a><a style='color: rgb(204,204,0); text-decoration:none' href='/check_status'>esults</a></b>
                                </td>
                                <td align='center' valign='center' bgcolor='#999966' width='80'>
                                    <b>Help</b>
                                </td>
                                <td align='center' valign='center' bgcolor='#333333' width='80'>
                                    <b><a style='color: rgb(255,0,0); text-decoration:none' href='/tutorial'>T</a><a style='color: rgb(204,204,0); text-decoration:none' href='/tutorial'>utorial</a></b>
                                </td>
                                <td align='center' valign='center' bgcolor='#333333' width='80'><b><a style='color: rgb(255,0,0); text-decoration:none' href='/citation'>C</a><a style='color: rgb(204,204,0); text-decoration:none' href='/citation'>itation</a></b>
			        </td>
			    </tr>
                        </table>
                        <table bgcolor='#999966' cellpadding='10%'>
                            <tr>
                                <td>
                                    <center>
                                        <table width='100%' cellpadding='15%'>
                                            <tr>
                                                <td align='center' valign='center' bgcolor='#333333'>
                                                    <font size=6><b><font color='#ff0000'>miR</font><font color='#cccc00'>vestigator Framework Help</font></b></font>
                                                </td>
                                            </tr>
                                        </table>
                                        </br>
                                        <table cellpadding='15%' cellspacing=3 width='100%'>
                                            <tr>
                                                <td bgcolor='#000000' colspan=2>
                                                    <center><font color='#cccc00' size=4><b>What is the miRvestigator Framework?</b></font></center>
                                                </td>
                                            </tr>
                                            <tr>
                                                <td colspan=2 bgcolor='#333333'><font color='#ffffff'><p>The miRvestigator framework is designed to take as input a list of co-expressed genes and will return the most likely <a href='http://en.wikipedia.org/wiki/MicroRNA' style='color: rgb(204,204,0)' target='_blank'>miRNA</a> regulating these genes. It does this by searching for over-represented sequence motifs in the <a href="http://en.wikipedia.org/wiki/3\'_UTR" style='color: rgb(204,204,0)' target='_blank'>3' untranslated regions (UTRs)</a> of the genes using <a href='http://159.149.109.9/modtools/' style='color: rgb(204,204,0)' target='_blank'>Weeder</a> and then comparing this to the miRNA seed sequences in <a href='http://www.mirbase.org' style='color: rgb(204,204,0)' target='_blank'>miRBase</a> using our custom built <a href='http://github.com/cplaisier/miRvestigator/wiki/miRvestigator-hidden-Markov-Model-(HMM)' target='_blank' style='color: rgb(204,204,0)'>miRvestigator hidden Markov model (HMM)</a>.</p></font>
                                                </td>
                                            </tr>
                                            <tr>
                                                <td colspan=2></td>
                                            </tr>
                                            <tr>
                                                <td bgcolor='#000000' colspan=2>
                                                    <center><font color='#cccc00' size=4><b>Entering a Co-Expressed Gene List</b></font></center>
                                                </td>
                                            </tr>
                                            <tr>
                                                <td colspan=2 bgcolor='#333333'>
                                                    <font color='#ffffff'><b>What is gene Co-Expression?</b>
                                                        <p><a href='http://en.wikipedia.org/wiki/Post-transcriptional_regulation' style='color: rgb(204,204,0)' target='_blank'>Post-transcriptional regulation</a> by a <a href='http://en.wikipedia.org/wiki/MicroRNA' style='color: rgb(204,204,0)' target='_blank'>miRNA</a> is mediated through binding to complementary sequences in the <a href="http://en.wikipedia.org/wiki/3\'_UTR" style='color: rgb(204,204,0)' target='_blank'>3' untranslated regions (UTRs)</a> of transcripts and generates a biological context-specific signature of down-regulation for these transcripts, which is called a co-expression signature. Co-expression signatures observed in transcriptome studies are thus assumed to be of biological origins whereby the expression of genes are regulated by shared factors (transcription factors, miRNAs, genetics or environmental factors). There are many methods available to identify co-expression signatures, and the miRvestigator framework does not bias toward any particular method.</p>
                                                        <b>Do I need quantitative expression data to use miRvestigator?</b>
                                                        <p>The input for miRvestigator is a gene list (e.g. 79073, 900, 29957, 11167, 4154, 84270, 84061, 4001, 26503, 4086, 51585, 4734, 5873) and thus does not require quantitative expression data as input. However, it is assumed that some other information was integrated to identify that set of genes, such as gene co-expression, specific miRNA perturbation, etc.</p>

                                                    <b>What species are supported by miRvestigator?</b> 
                                                    <p>miRvestigator currently supports:
                                                    <ul> 
                                                        <li><a href='http://en.wikipedia.org/wiki/Caenorhabditis_elegans' style='color: rgb(204,204,0)' target='_blank'>C. elegans</a> - a free-living, transparent nematode (roundworm), about 1 mm in length, which lives in temperate soil environments.</li> 
                                                        <li><a href='http://en.wikipedia.org/wiki/Drosophila_melanogaster' style='color: rgb(204,204,0)' target='_blank'>D. melanogaster</a> - common fruit fly.</li> 
                                                        <li><a href='http://en.wikipedia.org/wiki/Gallus_gallus' style='color: rgb(204,204,0)' target='_blank'>G. gallus</a> - an ancestor of the domesticated chicken.</li> 
                                                        <li><a href='http://en.wikipedia.org/wiki/Homo_sapiens' style='color: rgb(204,204,0)' target='_blank'>H. sapiens</a> - man.</li> 
                                                        <li><a href='http://en.wikipedia.org/wiki/Mus_musculus' style='color: rgb(204,204,0)' target='_blank'>M. musculus</a> - domesticated house mouse.</li> 
                                                        <li><a href='http://en.wikipedia.org/wiki/Brown_rat' style='color: rgb(204,204,0)' target='_blank'>R. norvegicus</a> - domesticated brown rat.</li> 
                                                    </ul> 
                                                    If the species you study has miRNAs and is not on this list please e-mail us and we will do our best to add your species of interest to miRvestigator.</p>
                                                    <b>Can miRvestigator include viral miRNAs?</b>
                                                    <p>For those who study the effects of viral miRNAs on host gene expression we have added the ability to include viral miRNAs. By selecting the Yes radio button the viral miRNAs will be searched along with the host species miRNAs. Note that perfectly overlapping seed seqeunces of a viral miRNA with a host miRNA will be concatenated as we cannot separate these influences. (<b>NOTE:</b> All viral miRNAs are included so you will have to filter them for those that you are interested in, please consult <a href='http://www.mirbase.org/cgi-bin/browse.pl' style='color: rgb(204,204,0)' target='_blank'>miRBase</a> for the species codes.)</p>
                                                    <b>What if I don't have one of your supported gene identifiers (IDs)?</b>
                                                    <p>Currently the miRvestigator framework is able to take <a href='http://www.ncbi.nlm.nih.gov/gene' style='color: rgb(204,204,0)' target='_blank'>Entrez gene IDs</a>, <a href='http://www.ensembl.org' style='color: rgb(204,204,0)' target='_blank'>Ensembl gene IDs</a>, <a href='http://www.ncbi.nlm.nih.gov/RefSeq/process.html' style='color: rgb(204,204,0)' target='_blank'>RefSeq transcript IDs</a> and official gene symbols as input. To convert any other type of identifier to Entrez gene IDs please use the <a href='http://david.abcc.ncifcrf.gov/home.jsp' style='color: rgb(204,204,0)' target='_blank'>DAVID</a> bioinformatics resource <a href='http://david.abcc.ncifcrf.gov/conversion.jsp' style='color: rgb(204,204,0)' target='_blank'>gene ID conversion tool</a>. The DAVID gene ID conversion tool has a great <a href='http://david.abcc.ncifcrf.gov/helps/conversion.html' style='color: rgb(204,204,0)' target='_blank'>help page</a> please visit it if you have any questions.</p>
                                                    <b>What types of <a href='http://en.wikipedia.org/wiki/Delimiter' style='color: rgb(255,255,255)' target='_blank'>delimiters</a> can be entered into the form?</b>
                                                    <p>You can enter the gene IDs separated by commas, spaces, tabs or <a href='http://en.wikipedia.org/wiki/Newline' style='color: rgb(204,204,0)' target='_blank'>newlines</a>.</p>
                                                    <b>Loading and Using Sample Data</b> <p>Load the data by simply clicking the button with the corresponding miRNA on it, which will load the data and setup any parameters. Then click the submit button at the bottom of the page.</p>
                                                  <b>Where does the sample data come from?</b><p>For the <i>H. sapiens</i> (human) sample data are co-expression signatures reduced to Entrez ID gene lists from studies where a miRNA was experimentally perturbed and the resultant effect ascertained by transcriptome measurements (e.g. gene expression microarray). To provide a working sample dataset for other species where an experimentally perturbed datasets were not yet available we utilized TargetScan miRNA target mRNA predictions for miR-1.</p>
                                                    <ul><li><b><i>C. elegans</i></b>
                                                    <ul><li><a href='http://www.mirbase.org/cgi-bin/mature.pl?mature_acc=MIMAT0000003' style='color: rgb(204,204,0)' target='_blank'><b>cel-miR-1</b></a> - TargetScan 5.1 cel-miR-1 target gene predictions.<p><a href='http://www.ncbi.nlm.nih.gov/pubmed/15652477' target='_blank' style='color: rgb(255,255,204)'>Lewis BP, Burge CB, Bartel DP. Conserved seed pairing, often flanked by adenosines, indicates that thousands of human genes are microRNA targets.<i>Cell</i> <b>120(1)</b>:15-20 (2005).</a></p></li></ul></li>
                                                    <li><b><i>D. melanogaster</i></b>
                                                    <ul><li><a href='http://www.mirbase.org/cgi-bin/mature.pl?mature_acc=MIMAT0000105' style='color: rgb(204,204,0)' target='_blank'><b>dme-miR-1</b></a> - TargetScan 5.1 dme-miR-1 target gene predictions.<p><a href='http://www.ncbi.nlm.nih.gov/pubmed/15652477' target='_blank' style='color: rgb(255,255,204)'>Lewis BP, Burge CB, Bartel DP. Conserved seed pairing, often flanked by adenosines, indicates that thousands of human genes are microRNA targets.<i>Cell</i> <b>120(1)</b>:15-20 (2005).</a></p></li></ul></li>
                                                    <li><b><i>G. gallus</i></b>
                                                    <ul><li><a href='http://www.mirbase.org/cgi-bin/mature.pl?mature_acc=MIMAT0001127' style='color: rgb(204,204,0)' target='_blank'><b>gga-miR-1a</b></a> - TargetScan 5.1 gga-miR-1a target gene predictions.<p><a href='http://www.ncbi.nlm.nih.gov/pubmed/15652477' target='_blank' style='color: rgb(255,255,204)'>Lewis BP, Burge CB, Bartel DP. Conserved seed pairing, often flanked by adenosines, indicates that thousands of human genes are microRNA targets.<i>Cell</i> <b>120(1)</b>:15-20 (2005).</a></p></li></ul></li>
                                                    <li><b><i>H. sapiens</i></b>
                                                      <ul><li><a href='http://www.mirbase.org/cgi-bin/mature.pl?mature_acc=MIMAT0000416' style='color: rgb(204,204,0)' target='_blank'><b>hsa-miR-1</b></a> - The HEK293T human embryonic kidney cell line with FLAG-Ago2 was transfected with and without exogenous miRNA. Genes significantly enriched (1% local FDR) in the co-immunoprecipitates of the FLAG-Ago2 transfected cell lines where the miRNA was included were considered targets of the miRNA.<p><a href='http://www.ncbi.nlm.nih.gov/pubmed/18461144' target='_blank' style='color: rgb(255,255,204)'>Hendrickson, D.G., Hogan, D.J., Herschlag, D., Ferrell, J.E. & Brown, P.O. Systematic identification of mRNAs recruited to argonaute 2 by specific microRNAs and corresponding changes in transcript abundance. <i>PLoS ONE</i> <b>3</b>, e2126 (2008).</a></p></li>
                                                          <li><a href='http://www.mirbase.org/cgi-bin/mature.pl?mature_acc=MIMAT0000441' style='color: rgb(204,204,0)' target='_blank'><b>hsa-miR-9</b></a> - Only genes exceeding median expression in at least half of the experiments were assayed for differential expression. Differential expression (P-value &lt; 0.001) was required to be consistently observed at both the 12 hour and 24 hour time points after miRNA transfection in HeLa cells. This dataset was reanalyzed from data in GEO data repository GSE8501 using the specification in the publication stated above to identify perturbed miRNA target genes.<p><a href='http://www.ncbi.nlm.nih.gov/pubmed/17612493' target='_blank' style='color: rgb(255,255,204)'>Grimson, A. et al. MicroRNA targeting specificity in mammals: determinants beyond seed pairing. <i>Mol. Cell</i> <b>27</b>, 91-105 (2007).</a></p></li>
                                                          <li><a href='http://www.mirbase.org/cgi-bin/mature.pl?mature_acc=MIMAT0000069' style='color: rgb(204,204,0)' target='_blank'><b>hsa-miR-16</b></a> - Down-regulated genes were identified at 24 hours after miRNA transfection with consistent differential expression (P-value &lt; 0.01) in HCT116 and DLD-1 colon tumor cell lines hypomorphic for Dicer (HCT116 Dicerex5 and DLD-1 Dicerex5, respectively) 2. Consensus miRNA down-regulated transcripts were also required to be significantly down-regulated (P-value &lt; 0.05) at 6 hours post-transfection in the colon cancer HTC116 Dicerex5 cell line 2.<p><a href='http://www.ncbi.nlm.nih.gov/pubmed/17242205' target='_blank' style='color: rgb(255,255,204)'>Linsley, P.S. et al. Transcripts targeted by the microRNA-16 family cooperatively regulate cell cycle progression. <i>Mol. Cell. Biol</i> <b>27</b>, 2240-2252 (2007).</a></p></li>
                                                          <li><a href='http://www.mirbase.org/cgi-bin/mature.pl?mature_acc=MIMAT0000426' style='color: rgb(204,204,0)' target='_blank'><b>hsa-miR-132</b></a> - Only genes exceeding median expression in at least half of the experiments were assayed for differential expression. Differential expression (P-value &lt; 0.001) was required to be consistently observed at both the 12 hour and 24 hour time points after miRNA transfection in HeLa cells. This dataset was reanalyzed from data in GEO data repository GSE8501 using the specification in the publication stated above to identify perturbed miRNA target genes.<p><a href='http://www.ncbi.nlm.nih.gov/pubmed/17612493' target='_blank' style='color: rgb(255,255,204)'>Grimson, A. et al. MicroRNA targeting specificity in mammals: determinants beyond seed pairing. <i>Mol. Cell</i> <b>27</b>, 91-105 (2007).</a></p></li>
                                                          <li><a href='http://www.mirbase.org/cgi-bin/mature.pl?mature_acc=MIMAT0000434' style='color: rgb(204,204,0)' target='_blank'><b>hsa-miR-142-3p</b></a> - Only genes exceeding median expression in at least half of the experiments were assayed for differential expression. Differential expression (P-value &lt; 0.001) was required to be consistently observed at both the 12 hour and 24 hour time points after miRNA transfection in HeLa cells. This dataset was reanalyzed from data in GEO data repository GSE8501 using the specification in the publication stated above to identify perturbed miRNA target genes.<p><a href='http://www.ncbi.nlm.nih.gov/pubmed/17612493' target='_blank' style='color: rgb(255,255,204)'>Grimson, A. et al. MicroRNA targeting specificity in mammals: determinants beyond seed pairing. <i>Mol. Cell</i> <b>27</b>, 91-105 (2007).</a></p></li>
                                                          <li><a href='http://www.mirbase.org/cgi-bin/mature.pl?mature_acc=MIMAT0000454' style='color: rgb(204,204,0)' target='_blank'><b>hsa-miR-184</b></a> - Reported consistent down-regulation (log(FC) &lt; -1, P-value &lt; 0.05) of non-redundant genes in the glioma cell lines A172 and T98G 72 hours after miRNA transfection that also were predicted to be targets of this same miRNA by miRBase target database.<p><a href='http://www.ncbi.nlm.nih.gov/pubmed/19775293' target='_blank' style='color: rgb(255,255,204)'>Malzkorn, B. et al. Identification and functional characterization of microRNAs involved in the malignant progression of gliomas. <i>Brain Pathol</i> <b>20</b>, 539-550 (2010).</a></p></li></ul></li>
                                                      <li><b><i>M. musculus</i></b>
                                                      <ul><li><a href='http://www.mirbase.org/cgi-bin/mature.pl?mature_acc=MIMAT0000123' style='color: rgb(204,204,0)' target='_blank'><b>mmu-miR-1</b></a> - TargetScan 5.1 mmu-miR-1 target gene predictions.<p><a href='http://www.ncbi.nlm.nih.gov/pubmed/15652477' target='_blank' style='color: rgb(255,255,204)'>Lewis BP, Burge CB, Bartel DP. Conserved seed pairing, often flanked by adenosines, indicates that thousands of human genes are microRNA targets.<i>Cell</i> <b>120(1)</b>:15-20 (2005).</a></p></li>
                                                          <li><a href='http://www.mirbase.org/cgi-bin/mature.pl?mature_acc=MIMAT0000246' style='color: rgb(204,204,0)' target='_blank'><b>mmu-miR-122</b></a> - Experimentally derived mmu-miR-122 target genes using antagomirs.<p><a href='http://www.ncbi.nlm.nih.gov/pubmed/16258535' target='_blank' style='color: rgb(255,255,204)'>Krützfeldt J, Rajewsky N, Braich R, Rajeev KG, Tuschl T, Manoharan M, Stoffel M. Silencing of microRNAs in vivo with 'antagomirs'. <i>Nature</i>. 2005 Dec 1;<b>438(7068)</b>:685-9.</a></p></li></ul></li>
                                                      <li><b><i>R. norvegicus</i></b>
                                                      <ul><li><a href='http://www.mirbase.org/cgi-bin/mature.pl?mature_acc=MIMAT0003125' style='color: rgb(204,204,0)' target='_blank'><b>rno-miR-1</b></a> - TargetScan 5.1 rno-miR-1 target gene predictions.<p><a href='http://www.ncbi.nlm.nih.gov/pubmed/15652477' target='_blank' style='color: rgb(255,255,204)'>Lewis BP, Burge CB, Bartel DP. Conserved seed pairing, often flanked by adenosines, indicates that thousands of human genes are microRNA targets.<i>Cell</i> <b>120(1)</b>:15-20 (2005).</a></p></li></ul></li>
                                                      <li><b><i>H. sapiens infected with Kaposi-Sarcoma-Associated Herpes Virus</i></b>
                                                      <ul><li><a href='http://mirbase.org/cgi-bin/mature.pl?mature_acc=MIMAT0002183' style='color: rgb(204,204,0)' target='_blank'><b>kshv-miR-K12-2</b></a> - Predicted targets of KSHV.<p><a href='http://www.ncbi.nlm.nih.gov/pubmed/20628498' target='_blank' style='color: rgb(255,255,204)'>Laganà A, Forte S, Russo F, Giugno R, Pulvirenti A, Ferro A. Prediction of human targets for viral-encoded microRNAs by thermodynamics and empirical constraints. <i>J RNAi Gene Silencing</i>. 2010 May 24;<b>6(1)<b>:379-85.</a></p></li></ul></li></ul></font>
                                                </td>
                                            </tr>
                                            <tr>
                                                <td>
                                                </td>
                                            </tr>
                                            <tr>
                                                <td bgcolor='#000000' colspan=2>
                                                    <font color='#cccc00' size=4><center><b>Parameters for Weeder</b></center></font>
                                                </td>
                                            </tr>
                                            <tr>
                                                <td colspan=2 bgcolor='#333333'><font color='#ffffff'>
                                                    <b>What is Weeder?</b>
                                                    <p><a href='http://159.149.109.9/modtools/' style='color: rgb(204,204,0)' target='_blank'>Weeder</a> is an <a href='http://en.wikipedia.org/wiki/Enumerative' style='color: rgb(204,204,0)' target='_blank'>enumerative algorithm</a> used to identify over-represented <a href='http://en.wikipedia.org/wiki/Sequence_motif' style='color: rgb(204,204,0)' target='_blank'>sequence motifs</a> in the miRvestigator framework.</p>
                                                    <b>What are the motif sizes?</b>
                                                    <p>Weeder identifies both 6 base-pair (bp) and 8bp motifs for each run. Using this option it is possible to choose which motif size is subsequently used in the miRvestigator HMM is set by this option. Picking only one of the motif sizes will make the run faster. By default we have chosen to use 8bp motifs. (<b>Note:</b> It is possible that the 6bp motif and the 8bp motif will be different.)</p>
                                                    <b>What background model does Weeder use?</b>
                                                    <p>Weeder uses the background model to determine whether or not an oligo is enriched above background. Each different species has a different background model based upon the sequence composition of the specified region in that species. The Weeder documentation states that the "Default Weeder Model" should be sufficient for 3' UTRs even though it was built based up on sequences upstream from the start codon, which was consistent with our findings. Although in certain cases a model built upon the annotated 3' UTR sequences from UCSC was helpful. Therefore, by default the "Default Weeder Model" is chosen although it may be best to try the "3' UTR Specific Model" as well. (<b>Note:</b> It is likely that the 3' UTR background model would improve given more complete 3'UTR annotations.)</p>
                                                    <b>What target site quality threshold is best?</b>
                                                    <p>The way Weeder is setup to run in the miRvestigator framework it identifies motifs based upon allowing at max 2bp of non-complementarity. The canonical view in the field of miRNAs is that 7mer and 8mer of complementarity to the seed sequence are the most efficacious. If the target site quality threshold is set at 100% Identity then only perfect 8mer or 6mer of complementarity (depending on the motif size) are returned, and if the target site quality threshold is set at 95% identity this typically retrieves putative sites with 1bp non-complementarity to the motif. Therefore to be consistent with the canonical view of the activity of miRNAs we set the target site quality threshold by default to 95% identity. (<b>Note:</b> When using 6bp motifs the target quality threshold ought be set to 100% identity, and the efficacy of the 6bp predicted binding sites would need to be extensively tested experimentally.)</p></font>
                                                </td>
                                            </tr>
                                            <tr>
                                                <td>
                                                </td>
                                            </tr>
                                            <tr>
                                                <td bgcolor='#000000' colspan=2>
                                                    <font color='#cccc00' size=4><b><center><b>Parameters for miRvestigator HMM</b></center></font>
                                                </td>
                                            </tr>
                                            <tr>
                                                <td colspan=2 bgcolor='#333333'>
                                                    <font color='#ffffff'><b>What is the miRvestigator hidden Markov model (HMM)?</b>
                                                    <p>The miRvestigator <a href='http://en.wikipedia.org/wiki/Hidden_Markov_model' style='color: rgb(204,204,0)' target='_blank'>hidden Markov model (HMM)</a> is a method designed to take an over-represented <a href='http://en.wikipedia.org/wiki/Sequence_motif' style='color: rgb(204,204,0)' target='_blank'>sequence motif</a> (in this case from <a href='http://159.149.109.9/modtools' style='color: rgb(204,204,0)' target='_blank'>Weeder</a>) and compare it to all the miRNA seed sequences from <a href='http://www.mirbase.org' style='color: rgb(204,204,0)' target='_blank'>miRBase</a> using the <a href='http://en.wikipedia.org/wiki/Viterbi_algorithm' style='color: rgb(204,204,0)' target='_blank'>Viterbi algorithm</a>. The over-represented sequence motif is turned into a profile HMM and each seed sequence in miRBase is aligned to identify the best complementarity and a probability of complementarity computed using the Viterbi algorithm. Then a Complementarity p-value is calculated for each miRNA by comparing it to an exhaustive distribution of Viterbi probabilities.<p>
                                                    <b>What are the seed models?</b>
                                                    <p>Base-pairing models for the seed regions of a miRNA to the 3' UTR of target transcripts. The 8mer, 7mer-m8, and 7mer-a1 models are the canonical models of miRNA to mRNA base-pairing. The 6mer models are considered marginal models as they typically have a reduced efficacy and are more likely to occur by chance alone. By default all of the seed models are used. The seed models are described in this figure:<center><img src='/images/seedModels.gif' width='400' style='background: rgb(204,204,204)'></center></p>
                                                    <b>Model Wobble Base-Pairing?</b>
                                                    <p>The miRvestigator HMM can also model G:U wobble base-pairing which has been observed in miRNA to target transcript 3' UTR complementarity. The sequence motif is unlikely to be comprised completely of instances where a G:U wobble was used in the exact same spot. However, it may be possible and we do not want to exclude the possibility. Thus the miRvestigator framework can be enabled to model these G:U base pairings once the G or U nucleotide frequency in the sequence motif column passes a specified threshold. This threshold can be set with the "Min. Freq. of G or U". By default the wobble base-pairing is not modeled but if wobble base-pairing is enabled we recommend a threshold of 0.25 for the "Min. Freq. of G or U".</p></font>
                                                </td>
                                            </tr>
                                            <tr>
                                                <td>
                                                </td>
                                            </tr>
                                            <tr>
                                                <td bgcolor='#000000' colspan=2>
                                                    <center><font color='#cccc00' size=4><b>Submitting a Job</b></font></center>
                                                </td>
                                            </tr>
                                            <tr>
                                                <td bgcolor='#333333'>
                                                    <font color='#ffffff'>
                                                        <b>How many top miRNAs to return?</b>
                                                        <p>miRvestigator provides as output a list of miRNAs from miRBase that are sorted based on the complementarity between their seed sequence and the motif identified from the 3' UTRs of the input sequences. We save all the results out, but allow the user to decide how many to return:  Top 10, Top 25, Top 50, Top 75, Top 100 or All.</p>
                                                        <b>Notification via e-mail when job is finished? <font color='#ff0000'>(Optional)</font></b>
                                                        <p>miRvestigator allows users to be notified by e-mail when there job has finished processing. Included in the e-mail will be a link to the results page. Allowing users to submit multiple jobs and access the results at their leisure. Alternatively you can bookmark either the status page or results page for your run to return to collect your results. (<b>NOTE:</b> If you don't receive e-mails please check your e-mail address, spam filter and add <u>mirvestigator@systemsbiology.org</u> to your address book. Typically by adding an e-mail address to your address book will allow e-mails from that address to bypass the spam filter.)</p>
                                                    </font>
                                                </td>
                                            </tr>
                                            <tr>
                                                <td colspan=2></td>
                                            </tr>
                                            <tr>
                                                <td bgcolor='#000000' colspan=2>
                                                    <center><font color='#cccc00' size=4><b>Retrieving Results</b></font></center>
                                                </td>
                                            </tr>
                                            <tr>
                                                <td bgcolor='#333333'>
                                                    <font color='#ffffff'>
                                                        <b>What happens after I click submit?</b>
                                                        <p>After clicking submit you will be taken to a status page (see Example Status Page below) which can be bookmarked to return and collect results at a later time. As your miRvestigator job is processed the status page will be updated with information about the run. The submission parameters are listed below the updates.</p>
                                                        <p><center><b>Example Status Page:</b></br><img src="/images/status_page.gif" width="600"></center></p>
                                                        <p>After the job has finished processing you will be taken automatically to the results page (seed Example Results Page Below) with the results for your run. The results page can also be bookmarked so that you can return to view the results at a later time. Results will be kept for at least two months.</p>
                                                        <p><center><b>Example Results Page:</b></br><img src="/images/results_page.gif" width="600"></center></p>
                                                        <b>Do I have to stay on the status page in order to see the results?</b>
                                                        <p>No you do not. Either by bookmarking the status page or by giving your e-mail address you can easily retrieve your results at a later time. (<b>NOTE:</b> If you don't receive e-mails please check your e-mail address, spam filter and add <u>mirvestigator@systemsbiology.org</u> to your address book. Typically by adding an e-mail address to your address book will allow e-mails from that address to bypass the spam filter.)</p>
                                                        <b>Can I retrieve my results with just the job id?</b>
                                                        <p>The job ID is a long character string that is unique for each job. If you retain this you can click on the results link at the top of any page (except the results page) to access the results from a previous run.</p>
                                                    </font>
                                                </td>
                                            </tr>
                                            <tr>
                                                <td colspan=2></td>
                                            </tr>
                                            <tr>
                                                <td bgcolor='#000000' colspan=2>
                                                    <center><font color='#cccc00' size=4><b>miRvestigator Results</b></font></center>
                                                </td>
                                            </tr>
                                            <tr>
                                                <td bgcolor='#333333'>
                                                    <font color='#ffffff'>
                                                    <p><center><b>Example Results Page:</b></br><img src="/images/results_page.gif" width="600"></center></p>
                                                    <b>What is on the results page?</b>
                                                    <p>The results page has at the top a hidden table of the <b>Submission Parameters</b>, which can be expanded by clicking on the <b>[+]</b>. Next is a <b>Summary of the Results</b> for each motif, which contains a table that has key information about each motif. For each motif we display:
                                                    <ul>
                                                        <li><b>Motif</b> - a logo plot for the motif identified from the 3' UTR sequences of the input sequences. (<b>Note:</b> This is also a link to the table of complementary miRNAs.)</li>
                                                        &nbsp;</br>
                                                        <li><b>Top miRNA</b> - the top complementary miRNA as determined by the miRvestigator HMM. This is a link to the miRNA in miRBase.</li>
                                                        &nbsp;</br>
                                                        <li><b>Complementary Base-Pairing</b> - complementarity of the consensus motif to the miRNA seed sequence for the top complementary miRNA.</li>
                                                        &nbsp;</br>
                                                        <li><b>Complementarity P-Value</b> - significance of the complementarity between the over-represented sequence motif and the miRNA seed sequence. (<b>Note:</b> A perfectly complementary 8mer seed model is 1.5e-05, for a 7mer seed model 6.1e-05, and for a 6mer seed model 0.00024.)</li>
                                                        &nbsp;</br>
                                                        <li><b>% of Input Sequences with Site</b> - percent of input sequences that had a predicted binding site passing the target site quality threshold. (<b>Note:</b> This is a link to the table of motif sites.)</li>
                                                    </ul>
                                                    <p>Following the Summary of Results are the <b>Motif Complementary miRNAs</b> and <b>Motif Sites</b> tables. These two tables contain the results for the miRvestigator framework run. We describe both of these tables in detail in the next two sections.</p>
                                                    </font>
                                                </td>
                                            </tr>
                                            <tr>
                                                <td colspan=2></td>
                                            </tr>
                                            <tr>
                                                <td bgcolor='#000000' colspan=2>
                                                    <center><font color='#cccc00' size=4><b>miRvestigator Results: Motif Complementary miRNAs Table</b></font></center>
                                                </td>
                                            </tr>
                                            <tr>
                                                <td bgcolor='#333333'>
                                                    <font color='#ffffff'>
                                                        <p><center><b>Example Motif miRNA Complementarity Table:</b></br><img src="/images/top_miRNAs.gif" width="600"></center></p>
                                                        <b>Can this table be downloaded?</b>
                                                        <p>There is a download table as CSV link at the top of the complementary miRNAs table. Click on the link and open the CSV downloaded to your computer using your favorite spreadsheet application.</p>
                                                        <b>What do the columns mean?</b>
                                                        <p><ul>
                                                            <li><b>miRNA Name</b> = The name of the name(s) for the unique seed sequence. There may be more than one miRNA annotated for a unique seed sequence because they vary in the 3' terminus of the mature miRNA. Each miRNA is a link to it's entry on <a href="http://www.mirbase.org" style="color: rgb(204,204,0)" target="_blank">miRBase</a></li>
                                                            &nbsp;</br>
                                                            <li><b>miRNA Seed</b> = The sequence of the most complementary seed for the over-represented motif. The seed will be as long as the seed model described in the next column.</li>
                                                            &nbsp;</br>
                                                            <li><b>Seed Model</b> = Base-pairing models for the seed regions of a miRNA to the 3' UTR of target transcripts. The 8mer, 7mer-m8, and 7mer-a1 models are the canonical models of miRNA to mRNA base-pairing. The 6mer models are considered marginal models as they typically have a reduced efficacy and are more likely to occur by chance alone. By default all of the seed models are used. The seed models are described in this figure:</br>&nbsp;</br><center><img src="/images/seedModels.gif" width=400></center></li>
                                                            &nbsp;</br>
                                                            <li><b>Length of Complementarity</b> = The length of complementarity (or wobble if enabled) base-pairing between the sequence motif and the miRNA seed sequence.</li>
                                                            &nbsp;</br>
                                                            <li><b>Complementary Base-Pairing</b> = The complementarity of the over-represented sequence motif on top 5'&rArr;3' to the miRNA seed sequence given the seed model 3'&rArr;5'. (<b>Note:</b> <span style='background-color: rgb(255,255,255);'>&nbsp;<b><font color="#ff0000">|</font></b>&nbsp;</span> = a complementary, <span style='background-color: rgb(255,255,255);'>&nbsp;<b><font color="#0000ff">:</font></b>&nbsp;</span> = a wobble, <span style='background-color: rgb(255,255,255);'><font color="#000000">"</font> <font color="#000000">"</font></span> (space) = not complementary, and for the sequences <span style='background-color: rgb(255,255,255);'>&nbsp;<b><font color="#cccccc">-</font></b>&nbsp;</span> = a gapping at the start or end.)</li>
                                                            &nbsp;</br>
                                                            <li><b>Complementarity P-Value</b> = Significance of complementarity between the over-represented sequence motif and the miRNA seed sequence. (<b>Note:</b> A perfectly complementary 8mer seed model has a Complementarity p-value of 1.5e-05, for a 7mer seed model 6.1e-05, and for a 6mer seed model 2.4e-04.)</li>
                                                        </ul>
                                                        <b>What is considered good complementarity?</b> <p>This will depend upon your data and what downstream analysis you plan to do with it. But a good rule of thumb is that if you find perfect complementarity for a 7mer or 8mer (Complementarity P-Value = <font color="#ff0000"><b>6.1e-05</b></font> and <font color="#ff0000"><b>1.5e-05</b></font>; respectively) this is likely to be of interest. Follow up with experimental studies will help to determine the false discovery rate for your dataset.</p>                                                    
                                                    </font>
                                                </td>
                                            </tr>
                                            <tr>
                                                <td colspan=2></td>
                                            </tr>
                                            <tr>
                                                <td bgcolor='#000000' colspan=2>
                                                    <center><font color='#cccc00' size=4><b>miRvestigator Results: Motif Sites Table</b></font></center>
                                                </td>
                                            </tr>
                                            <tr>
                                                <td bgcolor='#333333'>
                                                    <font color='#ffffff'>
                                                    <p><center><b>Example Motif Sites Table:</b></br><img src="/images/sites_table.gif" width="600"></center></p>
                                                    <b>Can this table be downloaded?</b>
                                                    <p>There is a download table as CSV link at the top of the motif sites table. Click on the link and open the CSV downloaded to your computer using your favorite spreadsheet application. (<b>Note:</b> The downloaded table will not be filtered by target site quality threshold.</p>
                                                    <b>Where do these sites come from?</b>
                                                    <p>As part of the miRvestigator framework <a href="http://159.149.109.9/modtools/" style="color: rgb(204,204,0)" target="_blank">Weeder</a> provides predicted miRNA binging sites in the 3' untranslated regions (UTRs) of the analyzed genes. Predicted binding sites were split into three different similarity bins:  <font color="#ff0000">High quality</font> - &#8805; 95% similarity to the miRNA seed sequence (red), <font color="#cccc00">Medium quality</font> 95% &#8805; similarity &#8805; 90% to the miRNA seed sequence (yellow), and <font color="#00ff00">Fair quality</font> 90% &#8805; similarity &#8805; 85% to the miRNA seed sequence (green). These sites can be used to develop follow-up experiments such as luciferase reporter assays to validate the efficacy of these sites.</p>
                                                    <b>Are these sites filtered?</b>
                                                    <p>As part of the parameters for the run you can set the target site quality threshold to filter for perfect complementarity (100% Identity), perfect complementarity and 1bp discrepancies (95% Identity), or down to 2bp discrepancies. We recommend sticking with 100% or 95% identity thresholds as these are the most similar to the canonical view of the mechanistic action of miRNA binding through the seed sequence with 7bp or 8bp of similarity between the miRNA seed sequence and the target mRNA. If using a 6bp motif only use 100% identity threshold, and follow these up with extensive experimental validation.</p>
                                                    <b>What do the columns mean?</b>
                                                    <p><ul>
                                                        <li><b>Gene</b> = gene identifier from the input set that links to the mapped Entrez identifier page on NCBI.</li>
                                                        &nbsp;</br>
                                                        <li><b>Gene Symbol</b> = Official gene symbol if available.</li>
                                                        &nbsp;</br>
                                                        <li><b>Site</b> = The sequence for site identified by Weeder. If it is in square brackets indicates that the site is of lower similarity.</li>
                                                        &nbsp;</br>
                                                        <li><b>Start Relative to Stop Codon</b> = The 3' UTR begins following the stop codon (which is set at 0 base-pairs (bp)). Thus the values in this column describe the start of the site in bp after the stop codon.</li>
                                                        &nbsp;</br>
                                                        <li><b>% Similarity to Consensus Motif</b> = The similarity of the predicted site to the consensus motif is computed as a percentage. Predicted binding sites were split into three different similarity to consensus bins:  <font color="#ff0000">High quality</font> - &#8805; 95% similarity to the miRNA seed sequence (red), <font color="#cccc00">Medium quality</font> 95% &#8805; similarity &#8805; 90% to the miRNA seed sequence (yellow), and <font color="#00ff00">Fair quality</font> 90% &#8805; similarity &#8805; 85% to the miRNA seed sequence (green).</li>
                                                        &nbsp;</br>
                                                        <li><b>Minimum Free Energy (MFE) of mRNA-miRNA Duplex</b> = the minimum free energy (MFE) of duplexing for the reverse complement of the motif consensus sequence and putative target site sequences using the ViennaRNA package.</li>
                                                    </ul></p>
                                                    </font>
                                                </td>
                                            </tr>
                                            <tr>
                                                <td colspan=2></td>
                                            </tr>
                                            <tr>
                                                <td bgcolor='#000000' colspan=2>
                                                    <center><font color='#cccc00' size=4><b>Other Implementations of miRvestigator</b></font></center>
                                                </td>
                                            </tr>
                                            <tr>
                                                <td bgcolor='#333333'>
                                                    <font color='#ffffff'>
                                                        <p><b>Stand-alone miRvestigator for High-Throughput Screening of Motifs</b></p>
                                                        <p>This is a command-line program to identify miRNAs from a Position Specific Scoring Matrix. A PSSM is read from a user specified file and is then passed to miRvestigator which downloads the mature miRNA seed sequences from miRbase.org and compares these mature seed sequences against the PSSM. This is accomplished by converting the PSSM into a Hidden Markov Model (HMM), very similar to a profile HMM, and then using the Viterbi algorithm to simultaneously align and calculate a probability for the alignment of the miRNA seed to the HMM. A p-value is calculated by exhaustively comparing all potential miRNA seed sequences that could bind to 3' UTRs, and using simulations the p-value for the Viterbi probability is the optimal metric for gauging miRNA seed to PSSM similarity.</br><ul><li><a href="http://github.com/cplaisier/miRvestigator_standalone" style="color: rgb(204,204,0)" target="_blank">miRvestigator Stand-alone</a> - Please follow instructions on readme.</li></ul></p>
                                                    </p>
                                                    </font>
                                                </td>
                                            </tr>
                                        </table>
                                        </br>
                                        <table width='100%' cellpadding='5%'><tr><td bgcolor='#c0c0c0'><center>Need help? Please contact <font color='#0000ff'>cplaisier(at)systemsbiology.org</font> or <font color='#0000ff'>cbare(at)systemsbiology.org</font> if you have any questions, comments or concerns.<br>Developed at the <a href='http://www.systemsbiology.org' target='_blank' style='color: rgb(0,0,255)'>Institute for Systems Biology</a> in the <a href='http://baliga.systemsbiology.net/' target='_blank' style='color: rgb(0,0,255)'>Baliga Lab</a>.</center></td></tr></table>
                                </td>
                            </tr>
                        </table>
                    </td>
                </tr>
            </table>
        </center>
    </body>
    
